scripts/ec2cleanup.py

```diff
@@ -218,8 +220,30 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_bucket(provider_mgmt, bucket_name, output):
+    buckets = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        bucket_list = provider_mgmt.list_s3_bucket_names()
+        for bucket in bucket_list:
+            if bucket_name in bucket:
+                provider_mgmt.delete_s3_buckets(bucket_names=[bucket])
+                buckets.append([provider_name, bucket])
+                logger.info("Deleted S3 Buckets: %r", bucket)
+        else:
```

Comment:
REQUIRED: this is wrong. this logging statement will appear in every run even when bucket is found.